src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsnonassociationsapi/dto/NonAssociation.kt

```diff
@@ -100,6 +103,19 @@ data class PatchNonAssociationRequest(
   val comment: String? = null,
 )
 
+/**
+ * Request format to close a non-association between two prisoners
+ */
+@Schema(description = "Request to close a non-association")
+data class CloseNonAssociationRequest(
+  @Schema(description = "Reason for closing the non-association", required = true, example = "They are friends now")
+  val closureReason: String,
+  @Schema(description = "Date and time of the closure, if not provided will default to today's time", required = false, example = "2023-06-07", defaultValue = "now")
+  val dateOfClosure: LocalDateTime? = null,
```

Comment:
why would we want clients of the api to specify any time other than "now"? for sync/migrate, sure, we need to consider other times, but for this…?

Reply:
I think its useful for potential data corrections